tests/test_dimension.py

```diff
@@ -830,6 +831,98 @@ def test_implicit_dims(self):
 
         assert np.all(f.data == F)
 
+    def test_lowering(self):
+        """
+        Test the lowering of a ConditionalDimension's condition.
+        This test makes an Operator that should indexify and lower the condition
+        passed in the Conditional Dimension
+        """
+
```

Comment:
no blank lines between docstrings and first line of code in tests

Reply:
Fixed